catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -79,6 +79,16 @@
 
   private Boolean phonetics;
 
+  public String getCacheId() {
```

Comment:
What does `cacheId` mean in the context of `CqlRequest`?

Also, since there is already a 'secondary' associated ID in `CqlRequest` called `batchId`, I'm wondering if we should just replace it with a `Map<String, String>` that can hold the existing `batchId`, your new ID, and any other future IDs we need to pass along (e.g. `{"batch": "<UUID>", "cache": "<UUID>"}`.

Reply:
Essentially, we wanted a way to pass the notification ID itself with the request, so that any sources that cache results based on the notification ID could associate the two. We could use a map, thought that would require changing whatever uses `batchId` as well